specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.3-preview/keys.json

```diff
@@ -1127,38 +1127,38 @@
           }
         },
         "x-ms-examples": {
-          "GetRandomNumbers": {
-            "$ref": "./examples/RandomNumbers-example.json"
+          "GetRandomBytes": {
+            "$ref": "./examples/GetRandomBytes-example.json"
           }
         }
       }
     }
   },
   "definitions": {
-    "GetRandomNumbersRequest": {
+    "GetRandomBytesRequest": {
       "properties": {
-        "bytesLength": {
+        "count": {
           "type": "integer",
           "format": "uint32",
           "minimum": 1,
           "maximum": 128,
           "description": "The requested number of random bytes."
         }
       },
-      "description": "The get random numbers request object.",
+      "description": "The get random bytes request object.",
       "required": [
-        "bytesLength"
+        "count"
       ]
     },
-    "GetRandomNumbersResponse": {
+    "GetRandomBytesResponse": {
```

Comment:
I doubt we'd expose this model type directly, but if we wanted to, it should not start with a verb and ideally not end in "Response". Just `RandomBytes` as a response model would be best.

Reply:
I changed GetRandomBytesResponse to RandomBytes as you suggested.